pages/linux/slapt-get.md

```diff
@@ -0,0 +1,28 @@
+# slapt-get
+
+> An apt like system for Slackware package management.
+> Package sources need to be configured on the slapt-getrc file.
+
+- Update the list of available packages and versions (it's recommended to run this before other `slapt-get` commands):
+
+`slapt-get --update`
+
+- Install a package, or update it to the latest available version:
+
+`slapt-get --install {{package}}`
+
+- Remove a package:
+
+`slapt-get --remove {{package}}`
+
+- Upgrade all installed packages to their newest available versions:
+
+`slapt-get --upgrade {{package}}`
+
+- Locate packages of interest by the package name, disk set, or version:
```

Comment:
What's a _disk set_? Not a _Slackware_ user, so I'm unsure :P

Reply:
Slackware used to be distribuited as a set of iso images (multiple CDs) where each package category (packages for gnome, packages for kde, packages for developers)  were in a different disk. Now there is only one ISO, but the package categories remain and are still called disk sets.

Reply:
Ah, I see :-)